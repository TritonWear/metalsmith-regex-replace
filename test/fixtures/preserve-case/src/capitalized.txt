Mr. Clean is a clean dude.