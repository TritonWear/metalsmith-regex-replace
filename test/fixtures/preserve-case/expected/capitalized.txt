Mr. Peanut is a peanut dude.